### ISSUE TITLE
[BUG] Discrepancy in the Effort using daytraderApp

### ISSUE THREAD

**Issue by kpunwatk** [konveyor/analyzer-lsp#616]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Konveyor version

latest

### Priority

Major

### Current Behavior

analysis_on_dayTraderapp_openJdk21:
"source": "Source code + dependencies",

"target": ["OpenJDK 21"],

"appName": "dayTraderApp",

"New effort": 429-439

"Previous effort": 596

![Screenshot from 2024-06-05 14-01-25](https://github.com/konveyor/analyzer-lsp/assets/87791967/32baa86f-906d-43a9-885f-74095e6d706c)

### How Reproducible

Always (Default)

### Environment

```markdown
- **Analyzer Image SHA** 
- **JDTLS  base image SHA**
- **Provider Settings JSON**
```

**Comment by pranavgaikwad** [comment]
@kpunwatk This is a non-issue because we have improved the accuracy of searches...if you have a specific rule that isn't matching or producing a false positive. Happy to re-open this issue. Thank you!
